specification/splitio/resource-manager/SplitIO.Experimentation/preview/2024-03-01-preview/splitio.json

```diff
@@ -670,16 +640,64 @@
         "accessPolicy"
       ]
     },
+    "ExperimentationWorkspaceProvisioningState": {
+      "type": "string",
+      "description": "The experimentation workspace resource provisioning state",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Accepted",
+        "Created",
+        "Deleting"
+      ],
+      "x-ms-enum": {
+        "name": "ExperimentationWorkspaceProvisioningState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Succeeded",
+            "value": "Succeeded",
+            "description": "Resource has been created."
+          },
+          {
+            "name": "Failed",
+            "value": "Failed",
+            "description": "Resource creation failed."
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled",
+            "description": "Resource creation was canceled."
+          },
+          {
+            "name": "Accepted",
+            "value": "Accepted",
+            "description": "Accepted"
+          },
+          {
+            "name": "Created",
+            "value": "Created",
+            "description": "Created"
+          },
+          {
```

Comment:
Do you need a Deleted state or is that covered in Succeeded?

Reply:
The RP is behind Rpaas and will be deleted once delete async operation completes, so we don't have deleted state.